### PR TITLE
Ahmadelmorshedy branch

### PR DESCRIPTION
**Pull request by ahmadelmorshedy**
Adding new options under learning (arabic language)

### CODE REVIEW

**Comment by ahmadelmorshedy** [comment]
Please review this, @casey-collab

File: arabic/learn/whattolearn/cooking.md [hunk 1]
```diff
@@ -0,0 +1 @@
+ممكن أن تبدأ [بهذا](http://www.allrecipes.com)
```
Comment (casey-collab) [inline arabic/learn/whattolearn/cooking.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: arabic/learn/whattolearn/drawing.md [hunk 2]
```diff
@@ -0,0 +1 @@
+ممكن أن تبدأ [بهذا](http://www.learn-to-draw.com/)
```
Comment (casey-collab) [inline arabic/learn/whattolearn/drawing.md]: This file has also been added to a directory that already contained a page.

File: arabic/learn/whattolearn/other.md [hunk 3]
```diff
@@ -0,0 +1 @@
+حدد ما تريد أن تتعلم، واعرف عنه المزيد من [هنا](https://www.google.com)
```
Comment (casey-collab) [inline arabic/learn/whattolearn/other.md]: This file has also been added to a directory that already contained a page.

File: arabic/learn/whattolearn/photography.md [hunk 4]
```diff
@@ -0,0 +1 @@
+ممكن أن تبدأ [بهذا](http://learnbasicphotography.com/)
```
Comment (casey-collab) [inline arabic/learn/whattolearn/photography.md]: This file has also been added to a directory that already contained a page.

File: arabic/learn/whattolearn/programming.md [hunk 5]
```diff
@@ -0,0 +1 @@
+ممكن أن تبدأ [بهذا](https://www.udacity.com)
```
Comment (casey-collab) [inline arabic/learn/whattolearn/programming.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ahmadelmorshedy** [comment]
it is fixed now @casey-collab

**Comment by casey-collab** [comment]
Looks great! :shell:

**Comment by ahmadelmorshedy** [comment]
Thanks :+1:

**Pull request merged**